.github/workflows/package_deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Package and Deploy
+
+on:
+  push:
+    branches:
+      - feature/package-artifact
```

Comment:
is this only for test purpose or an oversight?

Reply:
Yes. Will be removed before merger.